app/components/course-stage-screencast-preview.hbs

```diff
@@ -20,7 +20,7 @@
     <div class="flex gap-3 mt-3">
       <div class="flex items-center">
         {{svg-jar "clock" class="w-4 text-gray-300 mr-1.5"}}
-        <div class="text-gray-600 text-xs">{{round (div @screencast.durationInSeconds 60)}} minutes</div>
+        <div class="text-gray-600 text-xs" data-test-screencasts-formatted-duration>{{format-time-duration @screencast.durationInSeconds}}</div>
```

Comment:
```suggestion
        <div class="text-gray-600 text-xs" data-test-screencasts-formatted-duration>{{format-duration @screencast.durationInSeconds}}</div>
```

Duration seems sufficient as a name - don't think we need the "time" prefix. 

For example, the underlying date-fns library has formatDuration, and then there's  https://api.rubyonrails.org/classes/ActiveSupport/Duration.html. Neither have the "time" prefix